FirebaseCoreDiagnostics.podspec

```diff
@@ -15,7 +16,7 @@ variable FIREBASE_DIAGNOSTICS set to 0.
 
   s.source           = {
     :git => 'https://github.com/firebase/firebase-ios-sdk.git',
-    :tag => 'CoreDiagnostics-' + s.version.to_s
+    :tag => 'FirebaseCoreDiagnostics-' + s.version.to_s
```

Comment:
Standard practice is not to include `Firebase` on the tag.

Reply:
Fixed all the tags, even for GDT.